src/common/services/remote-config/defaults.ts

```diff
@@ -33,8 +33,6 @@ export const remoteConfigStringDefaults: {
   // Audius user id
   [StringKeys.TRENDING_PLAYLIST_OMITTED_USER_IDS]: '51',
   // Remove when Rewards UI update is live
```

Comment:
Can remove this comment too to avoid confusion

Reply:
nice